### PR TITLE
[ISSUE #5617]  The Query Parameter Lose

### PR DESCRIPTION
**Pull request by MisterChangRay**
Please do not create a Pull Request without creating an issue first.

fix issue #5617 
修正问题 #5617
  
## What is the purpose of the change
History Version Details When you return to the history version list page, the query parameters for the list page are missing. View is returned with parameter error.

历史版本详情在返回到 历史版本列表页时 , 列表页的查询参数丢失。 查看是返回时携带参数错误。

## Brief changelog

更正查询参数为： historyGroup & historyDataId

change param name to historyGroup & historyDataId

## Verifying this change

XXXX

Follow this checklist to help us incorporate your contribution quickly and easily:

* [ ] Make sure there is a Github issue filed for the change (usually before you start working on it). Trivial changes like typos do not require a Github issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue.
* [ ] Format the pull request title like `[ISSUE #123] Fix UnknownException when host config not exist`. Each commit in the pull request should have a meaningful subject line and body.
* [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
* [ ] Write necessary unit-test to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add integration-test in [test module](https://github.com/alibaba/nacos/tree/master/test).
* [ ] Run `mvn -B clean package apache-rat:check findbugs:findbugs -Dmaven.test.skip=true` to make sure basic checks pass. Run `mvn clean install -DskipITs` to make sure unit-test pass. Run `mvn clean test-compile failsafe:integration-test`  to make sure integration-test pass.

### CODE REVIEW

**Review by haoyann — APPROVED** [review]
LGTM

**Comment by KomachiSion** [comment]
Please change the PR title. Format rule see the PR descrption.

**Comment by MisterChangRay** [comment]
> Please change the PR title. Format rule see the PR descrption.

OK, it has been modified

**Comment by plusmancn** [comment]
@MisterChangRay @KomachiSion it also has been fixed in #5566

File: console-ui/src/pages/ConfigurationManagement/HistoryDetail/HistoryDetail.js [hunk 1]
```diff
@@ -85,7 +85,7 @@ class HistoryDetail extends React.Component {
 
   goList() {
     this.props.history.push(
-      `/historyRollback?serverId=${this.serverId}&group=${this.group}&dataId=${this.dataId}&namespace=${this.tenant}`
+      `/historyRollback?serverId=${this.serverId}&historyGroup=${this.group}&historyDataId=${this.dataId}&namespace=${this.tenant}`
```
[the remark this answers is not in the document]
Comment (plusmancn) [reply console-ui/src/pages/ConfigurationManagement/HistoryDetail/HistoryDetail.js:88]: I start a review, and I will undo the same changes in https://github.com/alibaba/nacos/pull/5566/files#diff-fd319372056fa9ca18696767948b3dcbced4a8e34225e849b10d551b529c8f0d

**Review by KomachiSion — APPROVED** [review]

**Pull request merged**